components/blog/Loader.tsx

```diff
@@ -0,0 +1,16 @@
+const Loader = () => {
+	return (
+		<div className="absolute inset-0 flex items-center justify-center z-50  bg-opacity-40 bg-gray-500">
+			<output>
+				<img
```

Comment:
Use `next/Image` component instead of `img`. 

`Image` component does quite a few optimizations and adds a few security measures. You will see a warning for using `img` when you run `npm run build`.

Reply:
done